tests/plugins/imagebase/features/link.js

```diff
@@ -410,6 +410,33 @@
 				}
 			} );
 		},
+
+		// #tp3298
+		'test unlink with data callback after it': function( editor, bot ) {
+			addTestWidget( editor );
+
+			testUnlinkCommand( {
+				bot: bot,
+				html: '<figure><a href="http://foo"><img src="%BASE_PATH%_assets/logo.png"></a></figure>',
+
+				callback: function( evt, widget ) {
+					assertUnlinkWidget( {
```

Comment:
This call makes this test case just a copy&pasta of `'test dialog ok listener is deleted when closing the dialog'` either merge it into one, or remove this assert to reduce responsibility.

I think the later is a better option.